docs/instruments/LORISLogicParser.md

```diff
@@ -0,0 +1,143 @@
+
+This ReadMe breaks down the different parts of the Parser and lists syntax rules.
+
+Note that this parser is made up of two separate components: a JS parser and a PHP parser, both of which use the same syntax detailed below.
+
+# JS Use
+At the top of your JS file add `import { Evaluator } from 'Parser';` (change the path based on your directory location).
+<br>Call `Evaluator(LOGIC_STRING, SCOPE)` to evaluate an equation.
+
+# PHP Use
+At the top of your PHP file add `include 'Parser/php/evaluator.php';`. 
+<br>Call the evaluator with `Evaluator::evaluate($equation, $this->scope);`.
+
+# Development
+To add, delete, or edit functions, simply edit `js/Functions.js` and `php/functions.php`.
+
+If you wish to make changes to the syntax, first be sure you cannot implement the same functionality with a function call. If not, proceed with the instructions below.
+
+NOTE: Please make sure all changes are mirrored on both the JS and PHP sides of the parser. Discrepancies will cause more headaches than you or I wish to deal with. ALL changes to ANY part of the parser on the JS or PHP side MUST be unit tested! Integration tests to come.
+
+### Prerequisites for Development
+
+ * [Jison](jison.org) (only required for changes in syntax)
+
+ Note that end users do not require Jison, only the Jison-generated parser file.
+
+### Syntax and Operator Changes
+Changing syntax (or adding unary/binary operators) requires changes to `jison/logicParser.jison` as well as `php/lexer.php` and `php/parser.php`. See specifics below.
+
+### Jison/JS Grammar Specifics
+In `logicParser.jison`, tokens are defined at the top, precedence and assertions are set below that, 
+and finally the grammar itself is defined below that.
+The output is a simple object defining a tag, operation, and arguments. `tag` indicates the operation type to be handled in `Evaluator.js`. `op` indicates the operation to be executed (defined in `Functions.js`). Lastly, `args` defines the arguments of the operation as objects, which allows nested operations.
+See [Jison documentation](jison.org) for grammar and Flex pattern matching specifications.
+
+After your changes are made run `jison jison/logicParser.jison` and replace `js/logicParser.js` with the output file.
+
+### PHP Grammar Specifics
+Tokens are defined in `lexer.php`. Precedence and assertions are defined by the order of parsing functions in `parser.php`. The grammar is defined in `lexer.php`. Similarly to JS, an array defining `tag`, `op`, and `args` is output to `evaluator.php`.
+
+### Evaluator Changes
+To add new types of operations, add a case to the switch statement in `js/Evaluator.js` and `php/evaluator.php`.
+
+### Unit Testing
+JS tests can be added to `Loris/test/js-tests/Parser.test.js`. Run tests with `npm run tests:unit:js:watch`.
+
+PHP tests can be added to `Loris/test/unittests/ParserTest.php`. Run tests with `Loris/vendor/bin/phpunit --configuration phpunit.xml --testsuite 'PHPParserTest'`.
```

Comment:
Most of these sections don't belong here, since this is just here to define the language for the Z-JSON format to reference. The above descriptions of how to change things shouldn't be in the spec documentation.